diff -ruN a/pyqt_setuptools.py b/pyqt_setuptools.py
--- a/pyqt_setuptools.py	2023-02-26 02:32:33.676218960 +0100
+++ b/pyqt_setuptools.py	2023-02-26 02:33:34.996394283 +0100
@@ -226,8 +226,8 @@
     def _sip_compile(self, source, sip_builddir):
         """Compile sip file to sources."""
 
-        pyqt5_include_dir = os.path.join(
-            get_path('platlib'), 'PyQt5', 'bindings')
+        pyqt5_include_dir = '/app/lib/python3.11/site-packages/PyQt5/bindings'
+
         pyqt5_toml = os.path.join(pyqt5_include_dir, 'QtCore', 'QtCore.toml')
         with open(pyqt5_toml, 'rb') as fin:
             pyqt5_cfg = tomli.load(fin)
